packages/grid/src/BaseCell.tsx

```diff
@@ -12,6 +12,49 @@ export function getCellId<T>(rowKey: string, column: GridColumnModel<T>) {
   return `R${rowKey}C${column.info.props.id}`;
 }
 
+const icons = {
```

Comment:
I feel these may be used in other components for validation, may be good separating them out somewhere...? can be done in the future though

Reply:
They might be used in FF but not sure. I prefer to keep things co-located until it doesn't make sense anymore. But yeah, as soon as another component needs this we should break them out.

Reply:
I wonder if it's still worth putting these in a separate file at least?

Reply:
✅